src/openrct2/interface/Widget.h

```diff
@@ -60,6 +61,62 @@ enum
 
 constexpr uint8_t SCROLLBAR_WIDTH = 10;
 
+constexpr rct_widget MakeWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
+    rct_string_id tooltip = STR_NONE)
+{
+    rct_widget out = {};
+    out.left = origin.x;
+    out.right = origin.x + size.width - 1;
+    out.top = origin.y;
+    out.bottom = origin.y + size.height - 1;
+    out.type = type;
+    out.colour = colour;
+    out.content = content;
+    out.tooltip = tooltip;
+
+    return out;
+}
+
+constexpr rct_widget MakeRemapWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
+    rct_string_id tooltip = STR_NONE)
+{
+    rct_widget out = MakeWidget(origin, size, type, colour, content, tooltip);
+
+    // TODO: implement this as a constant.
+    out.content |= IMAGE_TYPE_REMAP;
+
+    return out;
```

Comment:
Can be simplified to:
```suggestion
    return MakeWidget(origin, size, type, colour, IMAGE_TYPE_REMAP | content, tooltip);
```